### DISCUSSION TITLE
blog/20240111/

### DISCUSSION THREAD

**Discussion by giscus-x** [xaoxuu/blog-comments#40] [Announcements]
# blog/20240111/

近期（指一个月前）终于有时间偿还前辈留下的技术债了，下面分享一下经验和心得。

https://xaoxuu.com/blog/20240111/

**Comment by rebron1900** [comment]
抛出异常会占用很多运算速度，警告这些也会，尽量还是处理完，毕竟看着也闹心不是。

**Reply by xaoxuu** [reply]
是啊，改这些警告的过程中也解决掉了很多崩溃隐患。

**Reply by xaoxuu** [reply]
话说大佬是怎么在茫茫互联网找到我的小站的？

**Comment by W4J1e** [comment]
警告太多看着也闹心，老代码真的成为了“屎山”不如有时间推倒重来更方便哈哈哈哈。

**Reply by xaoxuu** [reply]
公司不太可能会给时间专门做重构的

**Comment by penndu** [comment]
新版本Volantis新版本Volantis新版本Volantis~

**Reply by xaoxuu** [reply]
在写了在写了 ![image](https://github.com/xaoxuu/blog-comments/assets/16400144/49801943-1ee8-4db2-aac9-773871fe8f24)

**Reply by penndu** [reply]
还得催还得催~

**Reply by xaoxuu** [reply]
杜老师着急的话可以先换 stellar 用用看，反正标签组件的写法 v6 都会跟 stellar 用一套的，为了方便两个主题的用户随时切换。

**Reply by penndu** [reply]
一直想换到stellar来着，不够时间上一直不允许，年底了事就多，过年期间应该会有时间尝试切换！

**Comment by StationChnqoo** [comment]
``` java
printf("大佬儿这个主题简直是太强了 😄")
```
